tests/dragonfly/instance.py

```diff
@@ -279,6 +291,8 @@ def create(self, **kwargs) -> DflyInstance:
         args = {**self.args, **kwargs}
         args.setdefault("dbfilename", "")
         args.setdefault("use_zset_tree", None)
+        vmod = "dragonfly_connection=1,accept_server=1,listener_interface=1"
+        args.setdefault("vmodule", vmod)
```

Comment:
I suggest also adding `main_service=1` and also replication related files: rdb_save=1,replica=1